File: result/repair/transcoder/python2java/java_repair/MAXIMUM_SUBARRAY_SUM_ARRAY_CREATED_REPEATED_CONCATENATION.java
  int f_gold ( int [ ] a , int n , int k ) {
  int maxSoFar = - 2147483648 ;
  int maxEndingHere = 0 ;
  for ( int i = 0 ;
  i < n * k ;
  i ++ ) {
    maxEndingHere = maxEndingHere + a [ i % n ] ;
    if ( maxSoFar < maxEndingHere ) maxSoFar = maxEndingHere ;
    if ( maxEndingHere < 0 ) maxEndingHere = 0 ;
  }
  return maxSoFar ;
}

File: result/translate/starchat/java2python/python/MAXIMUM_SUBARRAY_SUM_ARRAY_CREATED_REPEATED_CONCATENATION.py
def f_gold(a, n, k):
        max_so_far = 0
        INT_MIN = sys.maxsize
        max_ending_here = 0
        for i in range(0, n * k):
            max_ending_here = max_ending_here + a[i % n]
            if max_so_far < max_ending_here:
                max_so_far = max_ending_here
            if max_ending_here < 0:
                max_ending_here = 0
        return max_so_far